allennlp/data/token_indexers/pretrained_transformer_mismatched_indexer.py

```diff
@@ -58,19 +58,15 @@ def count_vocab_items(self, token: Token, counter: Dict[str, Dict[str, int]]):
     @overrides
     def tokens_to_indices(self, tokens: List[Token], vocabulary: Vocabulary) -> IndexedTokenList:
         orig_token_mask = [1] * len(tokens)
-        tokens, offsets = self._intra_word_tokenize(tokens)
+        tokens, offsets = self._intra_word_tokenize(tokens)  # type: ignore
```

Comment:
This is a private method only called by this class.  Why do you need to ignore types?  You can just give the right type signature to the private method.  Is it because you're changing the type of `tokens` potentially?

Reply:
Yes I think so -- sorry this wasn't what we were talking about above.